Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2492,6 +2493,31 @@ ShouldOpenExternalURLsPolicy DocumentLoader::shouldOpenExternalURLsPolicyToPropa
     return ShouldOpenExternalURLsPolicy::ShouldNotAllow;
 }
 
+// https://www.w3.org/TR/css-view-transitions-2/#navigation-can-trigger-a-cross-document-view-transition
+bool DocumentLoader::navigationCanTriggerCrossDocumentViewTransition(Document& oldDocument)
+{
+    // implementation-defined navigation experience
```

Comment:
```suggestion
    // FIXME: Consider adding implementation-defined navigation experience step.
```

The current comment is a bit cryptic, I would either remove it, or reword it as a FIXME